benchmarks/nemo/tracer_advection/original/README.md

```diff
@@ -46,6 +51,11 @@ To use the NVIDIA compiler, OpenMP offload and managed memory:
 Scripts to do this for various compilers may be found in the `compiler_setup`
 directory at the root of this repository.
 
+Additionally, the optional `ENABLE_NVIDIA_PROFILE=yes` flag can be set to enable
+GPU offloading versions to generate instrumented code, this will require the
+`PSYCLONE_NVIDIA_LIB_DIR` to be set up to reference the path to PSyclone's nvidia
+profiling lib.
```

Comment:
"lib (see https://psyclone.readthedocs.io/en/stable/profiling.html#interface-to-third-party-profiling-tools)."

Reply:
Done